src/json/from_json.cr

```diff
@@ -231,13 +231,13 @@ def NamedTuple.new(pull : JSON::PullParser)
 
     {% for key, type in T %}
       if %var{key.id}.nil? && !%found{key.id} && !{{type.nilable?}}
-        raise JSON::ParseException.new("Missing json attribute: {{key}}", *location)
+        raise JSON::ParseException.new("Missing json attribute: #{{{key.id.stringify}}}", *location)
```

Comment:
Why `#{}`?

Reply:
If you just do `{{key}}` it would output `"Missing yaml attribute: xyz "foo" b-23"`, which breaks because the inner quotes are not escaped.  Using `#{}` ensures those quotes are escaped.

Technically this is a breaking change now that I think about given it would have previously been say `"Missing yaml attribute: x"` but now it would be `"Missing yaml attribute: "x""`.  However I don't see a way around it.

Reply:
Error messages are not guaranteed to be stable, nothing should prevent improvements there.